src/main/drivers/bus_spi.c

```diff
@@ -259,10 +261,17 @@ bool spiBusTransfer(const busDevice_t *bus, const uint8_t *txData, uint8_t *rxDa
 
 void spiSetDivisor(SPI_TypeDef *instance, uint16_t divisor)
 {
-#define BR_CLEAR_MASK 0xFFC7
+#define BR_BITS ((BIT(5) | BIT(4) | BIT(3)))
+
+    // SPI2 and SPI3 are always on APB1/AHB1 which PCLK is half that of APB2/AHB2.
```

Comment:
SPI4 is running on AHB2? That's weird, I would like to know the rationale behind peripheral number arrangement, e.g. why TIM1 and 8 are advanced but 2-7 are not :-)